desktop/app/app-state.js

```diff
@@ -0,0 +1,195 @@
+//
+// This is modified from https://github.com/mawie81/electron-window-state
+//
+
+import electron from 'electron'
+import fs from 'fs'
+import path from 'path'
+import jsonfile from 'jsonfile'
+import deepEqual from 'deep-equal'
+
+export default class AppState {
```

Comment:
Can we make any non public methods have the underscore prefix (i.e. `_privateMethod () {...`